src/compute/servers.rs

```diff
@@ -801,6 +807,28 @@ impl NewServer {
         self.add_port(port);
         self
     }
+
+    /// Use this user-data for the new server.
+    pub fn set_user_data(&mut self, user_data: String) {
```

Comment:
I think I wrote a macro for creating such set_XXX/with_XXX pairs, please check other services.

Reply:
There's creation_inner_field which sets self.inner.$name - I've added creation_field which skips the .inner.